force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -163,9 +201,9 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer generateRandomNumber() {
         // Use Math class to generate number between 50 and 100
-
+        Double a = Math.random() * (100 - 50) + 50;
```

Comment:
This will never return 100. Here's the general template for generating an random integer:
```
Integer randomNum = (Integer) (Math.random() * (max - min + 1)) + min;
Integer randomNum = (Integer) (Math.random() * (100 - 50 + 1)) + 50;
Integer randomNum = (Integer) (Math.random() * 51) + 50;
```